NPE when switching from offline -> online

Slide version: 5.5.1
Android version: v6.0.1

Steps to Reproduce:

1. Open the app with no internet connection
2. Switch internet back ON.
3. App Crashes with a NPE

Here's the stack trace for the record:
```
FATAL EXCEPTION: AsyncTask #4
Process: me.ccrama.redditslide.debug, PID: 30467
java.lang.RuntimeException: An error occurred while executing doInBackground()
    at android.os.AsyncTask$3.done(AsyncTask.java:309)
    at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:354)
    at java.util.concurrent.FutureTask.setException(FutureTask.java:223)
    at java.util.concurrent.FutureTask.run(FutureTask.java:242)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
    at java.lang.Thread.run(Thread.java:818)
 Caused by: java.lang.NullPointerException: Attempt to invoke 
virtual method 'okhttp3.OkHttpClient net.dean.jraw.http.OkHttpAdapter.getNativeClient()' 
on a null object reference
    at me.ccrama.redditslide.Authentication$1.doInBackground(Authentication.java:53)
    at me.ccrama.redditslide.Authentication$1.doInBackground(Authentication.java:50)
    at android.os.AsyncTask$2.call(AsyncTask.java:295)
    at java.util.concurrent.FutureTask.run(FutureTask.java:237)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113) 
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588) 
    at java.lang.Thread.run(Thread.java:818) 
```

--- a/app/src/main/java/me/ccrama/redditslide/Authentication.java
+++ b/app/src/main/java/me/ccrama/redditslide/Authentication.java
@@ -50,7 +50,7 @@ public class Authentication {
         new AsyncTask<Void, Void, Void>() {
             @Override
             protected Void doInBackground(Void... params) {
-                if(httpAdapter.getNativeClient() != null)
+                if(httpAdapter != null && httpAdapter.getNativeClient() != null)
                 httpAdapter.getNativeClient().connectionPool().evictAll();
                 return null;
             }

